src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -53,6 +57,7 @@ const RenderChildDashboard = props => {
               className="child-dash-img"
               src={change_your_avatar}
               alt="Change Your Avatar Button"
+              onClick={handleResults}
```

Comment:
I see that if the "Change your Avatar" box is clicked, it will handleResults and take user to Results page. In the future we can edit the text to direct user to "See the Results!". We generate the results on the moderator screen, so this shouldn't be a problem. :-) Just making a note of it for the future.